File: tema/src/main/java/ro/mpp/domain/Ticket.java
package ro.mpp.domain;

public class Ticket extends Entity<Long>{
    private int flightId;
    private String clientName;
    private String clientAddress;
    private String touristsNames;

    public Ticket(int flightId, String clientName, String clientAddress, String touristsNames) {
        this.flightId = flightId;
        this.clientName = clientName;
        this.clientAddress = clientAddress;
        this.touristsNames = touristsNames;
    }

    public int getFlightId() {
        return flightId;
    }

    public void setFlightId(int flightId) {
        this.flightId = flightId;
    }

    public String getClientName() {
        return clientName;
    }

    public void setClientName(String clientName) {
        this.clientName = clientName;
    }

    public String getClientAddress() {
        return clientAddress;
    }

    public void setClientAddress(String clientAddress) {
        this.clientAddress = clientAddress;
    }

    public String getTouristsNames() {
        return touristsNames;
    }

    public void setTouristsNames(String touristsNames) {
        this.touristsNames = touristsNames;
    }
}

File: Anul_2/Semestrul_2/Medii de proiectare si programare/mpp-proiect-csharp-andrea-muresan/TravelAgencyProject/Model/Ticket.cs
using System;

namespace Model
{
    [Serializable]
    public class Ticket : Entity<int>
    {
        public Flight Flight { get; set; }
        public string ClientName { get; set; }
        public string ClientAddress { get; set; }
        public string ClientsNames { get; set; }
        public int NoSeats { get; set; }

        public Ticket(Flight flight, string clientName, string clientAddress, string clientsNames, int noSeats)
        {
            this.Flight = flight;
            this.ClientName = clientName;
            this.ClientAddress = clientAddress;
            this.ClientsNames = clientsNames;
            this.NoSeats = noSeats;
        }
    }
}